src/Parser/Core/Modules/SpellHistory.js

```diff
@@ -60,7 +81,10 @@ class SpellHistory extends Analyzer {
   }
   on_toPlayer_updatespellusable(event) {
     const spellId = event.ability.guid;
-    this._append(spellId, event);
+    this._append(spellId, this._addTimeWaitingOnGCD(event));
```

Comment:
Nit pick: this suggests `_addTimeWaitingOnGCD` is a pure function while it mutates event. I think it would be cleaner if this were two lines without changing the value of `event`.